File: LC-1/LC-1-1.cpp
#include <vector>

using namespace std;

class Solution {
public:
    /**
     * Concepts: Brute Force
     * Find out each pair of element in the list that sum is equal to target
     */
    vector<int> twoSum(vector<int>& nums, int target) {
        for (int i = 0; i < nums.size() - 1; ++i) {
            for (int j = i + 1; j < nums.size(); ++j) {
                if (nums[i] == target - nums[j]) {
                    return vector<int>{i, j};
                }
            }
        }
        return vector<int>{};
    }
};

File: LC-1/LC-1-1.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */

var twoSum = function(nums, target) {
    /**
     * Concepts: Brute Force
     * Find out each pair of element in the list that sum is equal to target
     */
    
    if (nums == null || nums.length === 0) {
        return nums;
    }
    for (let i = 0; i < nums.length - 1; ++i) {
        for (let j = i + 1; j < nums.length; ++j) {
            if (nums[i] + nums[j] == target) {
                return [i, j];
            }
        }
    }
    return {};
};